src/app/components/book-dialog/add/add-dialog.component.html

```diff
@@ -0,0 +1,8 @@
+<h1 mat-dialog-title class="c-dialog__title">Add book</h1>
+<div mat-dialog-content>
+    <app-book-form [action]="action" [dialogRef]="dialogRef" (newBook)="getNewBook($event)" (formInvalid)="getFormStatus($event)" [dialogClosed]="dialogClosed"></app-book-form>
+</div>
+<div mat-dialog-actions class="c-dialog__actions fx fx-spacebetween">
+    <button mat-raised-button (click)="onNoClick()">Discard</button>
```

Comment:
We do have a directive `mat-dialog-close` for a cases when we need to simply close the dialog